docs/src/examples/03-parsimonious-flux-balance.jl

```diff
@@ -0,0 +1,99 @@
+
+# # Parsimonious flux balance analysis
+
+# We will use [`parsimonious_flux_balance`](@ref) and
+# [`minimize_metabolic_adjustment`](@ref) to find the optimal flux
+# distribution in the *E. coli* "core" model.
+#
+# TODO pFBA citation
+
+# If it is not already present, download the model and load the package:
+import Downloads: download
+
+!isfile("e_coli_core.json") &&
+    download("http://bigg.ucsd.edu/static/models/e_coli_core.json", "e_coli_core.json")
+
+# next, load the necessary packages
+
+using COBREXA
+
+import JSONFBCModels
+import Clarabel # can solve QPs
+
+model = load_model("e_coli_core.json") # load the model
+
+# Use the convenience function to run standard pFBA on
+
+vt = parsimonious_flux_balance(model, Clarabel.Optimizer; modifications = [silence])
+
+# Or use the piping functionality
+
+model |> parsimonious_flux_balance(Clarabel.Optimizer; modifications = [silence])
+
+@test isapprox(vt.objective, 0.87392; atol = TEST_TOLERANCE) #src
+@test sum(x^2 for x in values(vt.fluxes)) < 15000 #src
+
+#=
+
+# Alternatively, you can construct your own constraint tree model with
+# the quadratic objective (this approach is much more flexible).
+
+ctmodel = fbc_model_constraints(model)
+ctmodel *= :l2objective^squared_sum_objective(ctmodel.fluxes)
+ctmodel.objective.bound = 0.3 # set growth rate # TODO currently breaks
+
+opt_model = optimization_model(
+    ctmodel;
+    objective = ctmodel.:l2objective.value,
+    optimizer = Clarabel.Optimizer,
+    sense = Minimal,
```

Comment:
I don't like this `Minimal`. Since we are using JuMP's `set_optimizer_attribute` etc., we should use JuMP's `Min` and `Max`

Reply:
we're renaming JuMP's symbols to ours (people don't import them from JuMP!) so we shouldn't also force them to import parameters.